Formula/a/allure.rb

```diff
@@ -26,7 +26,6 @@ def install
     # Remove all windows files
     rm_f Dir["bin/*.bat"]
 
-    prefix.install_metafiles
     libexec.install Dir["*"]
```

Comment:
I'd be cautious about this ordering: `Dir["*"]` will pick these up and move it to libexec instead

Reply:
yeah, I just tested, it works fine, here is the diff btw the bottle and local source build, https://www.diffchecker.com/0leLV4CD/